app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -129,31 +165,47 @@ export default {
         sku: null,
         price: '',
         currency: 'CLP',
-        quantity: '',
-        measure: '',
+        ingredient_measures_attributes: [],
       },
+      showingMeasureModal: false,
+      usedUnits: [],
     };
   },
-
+  methods: {
+    AddMeasure() {
+      const last_item = this.form.ingredient_measures_attributes[this.form.ingredient_measures_attributes.length -1];
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Identifier 'last_item' is not in camel case                       camelcase